Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/GroupIBTIA.py

```diff
@@ -2272,21 +2327,32 @@ def main():
         incidents_first_fetch = params.get("first_fetch", "3 days").strip()
         requests_count = int(params.get("max_fetch", 3))
 
-        exclude_combolist = params.get("exclude_combolist", False)
+        combolist = params.get("combolist", False)
+        unique = params.get("unique", False)
         enable_probable_corporate_access = params.get("enable_probable_corporate_access", False)
+        limit_param = params.get("limit", 100)
+        limit = int(limit_param)
 
         args = demisto.args()
         command = demisto.command()
-        LOG(f"Command being called is {command}")
+        demisto.debug(f"Command being called is {command}")
+        demisto.debug(
+            "[main] Parsed params: "
+            f"url={base_url}, proxy={proxy}, verify={verify_certificate}, "
+            f"hunting_rules={hunting_rules}, first_fetch={incidents_first_fetch}, max_fetch={requests_count}, "
+            f"collections={incident_collections}, combolist={combolist}, unique={unique}, "
+            f"enable_probable_corporate_access={enable_probable_corporate_access}, limit={limit}"
+        )
```

Comment:
no need for this, params are automatically printed in the logs with debug-mode=true